src/libraries/Microsoft.Extensions.Hosting/src/Internal/ConsoleLifetime.cs

```diff
@@ -79,6 +80,11 @@ private void OnApplicationStarted()
             Logger.LogInformation("Application started. Press Ctrl+C to shut down.");
             Logger.LogInformation("Hosting environment: {EnvName}", Environment.EnvironmentName);
             Logger.LogInformation("Content root path: {ContentRoot}", Environment.ContentRootPath);
+
+            if (Path.GetFullPath(Environment.ContentRootPath).Equals(Path.GetFullPath("."), StringComparison.InvariantCultureIgnoreCase))
```

Comment:
> Path.GetFullPath(".")

Envrionment.CurrentDirectory?

> InvariantCultureIgnoreCase

wrong on unix?

Reply:
> Envrionment.CurrentDirectory ?

Envrionment.CurrentDirectory when !DisableDefaults, and AppContext.BaseDirectory elsewhere

> InvariantCultureIgnoreCase
This project uses OrdinalIgnoreCase for string comparison so I'm using Ordinal to be unix compliant